fe/fe-core/src/main/java/com/starrocks/planner/FileScanNode.java

```diff
@@ -238,8 +238,10 @@ private void initParams(ParamCreateContext context)
             params.setMulti_column_separator(fileGroup.getColumnSeparator());
         }
         if (row_delimiter.length != 1) {
-            throw new UserException(
-                    "invalid row delimiter '" + fileGroup.getRowDelimiter() + "': must be a single character");
+            if (row_delimiter.length > 50) {
+                throw new UserException("the row delimiter is limited to a maximum of 50 bytes");
```

Comment:
The row delimiter exceeds 50 bytes.

Reply:
It should be emphasized that the maximum value is 50